lib/vmdb/loggers.rb

```diff
@@ -50,20 +53,34 @@ def self.create_logger(log_file_name, logger_class = VMDBLogger)
       configure_external_loggers
     end
 
-    private_class_method def self.create_container_logger
-      return unless ENV["CONTAINER"]
+    private_class_method def self.create_broadcast_logger
+      return create_container_logger if container_logger?
+      return create_journald_logger  if journald_logger?
+
+      nil
```

Comment:
why the guard clauses and the `nil`?

```suggestion
      if container_logger?
        create_container_logger
      elsif journald_logger?
        create_journald_logger
      end
```

Reply:
I had what you had originally, and went back and forth - for some reason the version I committed looked cleaner to me.